test/Interop/Cxx/modules/Inputs/module.modulemap

```diff
@@ -1,4 +1,4 @@
-module Namespace {
+module Namespaces {
```

Comment:
I renamed the module to avoid name collision: both the namespace and its containing module were called `Namespace`. This is a general Swift issue that can be worked around by passing `-module-interface-preserve-types-as-written` flag.